v0/pipeline/steps/ci/docker-step-info.yaml

```diff
@@ -2,10 +2,16 @@ title: DockerStepInfo
 allOf:
 - $ref: ../../steps/ci/ci-step-info.yaml
 - type: object
-  required:
-  - connectorRef
-  - repo
-  - tags
+  oneOf:
+   - required:
+      - connectorRef
+      - repo
+      - tags
+   - required:
+       - registryRef
+       - useHarnessRegistry
+       - repo
+       - tags
```

Comment:
today, are repo and tags required fields?

Reply:
yes, they are required fields

Reply:
yes, they both are required